config/initializers/inflections.rb

```diff
@@ -0,0 +1,16 @@
+# Be sure to restart your server when you modify this file.
```

Comment:
Whats this file about?

Reply:
Rails uses the inflector library for handling the pluralization of different strings. This file allows us to create our own custom grammar rules if we need to, we have to write our own inflections for our application.

These singular and pluralization are used by Rails to be smart enough to handle cases, like knowing that a `mice` table should store the `Mouse` model objects, and that the `MiceController` should be responsible for passing those objects.